bazel/bazel.go

```diff
@@ -229,7 +230,7 @@ func (b *bazel) Info() (map[string]string, error) {
 	b.WriteToStderr(false)
 	b.WriteToStdout(false)
 	stdoutBuffer, _ := b.newCommand("info")
-
+	log.Logf("Running 'bazel info'...")
```

Comment:
What would you think about starting a goroutine and only printing this out if it takes a non-trivial amount of time to execute?

Reply:
Good idea. I'll add that in.

Reply:
Let me know what you think of the changes or how you would implement it :)